.eslintrc.js

```diff
@@ -1,44 +0,0 @@
-/*---------------------------------------------------------------------------------------------
```

Comment:
Why it is bad to have eslint settings in the root?

Reply:
Moved configs in shared package